build-logic/build-update-utils/src/main/kotlin/gradlebuild/buildutils/tasks/UpdateReleasedVersions.kt

```diff
@@ -26,9 +26,11 @@ import org.gradle.api.provider.Property
 import org.gradle.api.tasks.Internal
 import org.gradle.api.tasks.TaskAction
 import org.gradle.util.GradleVersion
+import org.gradle.work.DisableCachingByDefault
 import java.io.File
 
 
+@DisableCachingByDefault(because = "Not worth caching")
 abstract class UpdateReleasedVersions : DefaultTask() {
```

Comment:
Similar to `UpdateAgpVersions`.

Reply:
This is different, since the current contents of the released versions file is also an input and this doesn't read data from the internet.